srt.go

```diff
@@ -164,3 +246,67 @@ func (s Subtitles) WriteToSRT(o io.Writer) (err error) {
 	}
 	return
 }
+
+func (l Line) srtBytes() (c []byte) {
+	for idx, li := range l.Items {
+		c = append(c, li.srtBytes()...)
+		// condition to avoid adding space as the last character.
+		if idx < len(l.Items)-1 {
+			c = append(c, []byte(" ")...)
+		}
+	}
+	c = append(c, bytesLineSeparator...)
+	return
+}
+
+func (li LineItem) srtBytes() (c []byte) {
+	// Get color
+	var color string
+	if li.InlineStyle != nil && li.InlineStyle.TTMLColor != nil {
+		color = *li.InlineStyle.TTMLColor
+	}
+
+	// Get bold/italics/underline
+	b := li.InlineStyle != nil && li.InlineStyle.WebVTTBold
+	i := li.InlineStyle != nil && li.InlineStyle.WebVTTItalics
+	u := li.InlineStyle != nil && li.InlineStyle.WebVTTUnderline
+
+	// Append
+	if color != "" {
+		c = append(c, []byte("<font color=\""+color+"\">")...)
+	}
+	if b {
+		c = append(c, []byte("<b>")...)
+	}
+	if i {
+		c = append(c, []byte("<i>")...)
+	}
+	if u {
+		c = append(c, []byte("<u>")...)
+	}
+	c = append(c, []byte(li.Text)...)
+	if u {
+		c = append(c, []byte("</u>")...)
+	}
+	if i {
+		c = append(c, []byte("</i>")...)
+	}
+	if b {
+		c = append(c, []byte("</b>")...)
+	}
+	if color != "" {
+		c = append(c, []byte("</font>")...)
+	}
+	return
+}
+
+func getAttribute(n *html.Token, key string) (string, bool) {
```

Comment:
Could you move this function to `subtitles.go`, rename it to `htmlTokenAttribute()` and sure arguments are `(t html.Token, key string) string` ?